course/sitemaps.py

```diff
@@ -1,12 +1,11 @@
-from django.contrib import sitemaps
 from django.urls.base import reverse
 from django.utils import timezone
 
+from lib.sitemaps import AplusSitemap
 from .models import CourseInstance, CourseModule
 
 
-class CourseStaticViewSitemap(sitemaps.Sitemap):
-    priority = 0.5
```

Comment:
I think this class should inherit `sitemaps.Sitemap` since it has a static priority. The `AplusSitemap` does not improve this case. It is not necessary to override the `priority()` method since the `Sitemap` base class supports class variables.